deepmedic/routines/training.py

```diff
@@ -24,382 +24,437 @@
+                               n_samples_per_subep_train,
+                               samplingTypeInstanceTraining,
+                               listOfFilepathsToEachChannelOfEachPatientTraining,
+                               listOfFilepathsToGtLabelsOfEachPatientTraining,
+                               listOfFilepathsToRoiMaskOfEachPatientTraining,
+                               paths_to_wmaps_per_sampl_cat_per_subj_train,
+                               pad_input_imgs,
+                               augm_img_prms,
+                               augm_sample_prms)
+    args_for_sampling_val = (log,
+                             "val",
+                             num_parallel_proc_sampling,
+                             run_input_checks,
+                             cnn3dWrapper,
+                             max_n_cases_per_subep_train,
+                             n_samples_per_subep_val,
+                             samplingTypeInstanceValidation,
+                             listOfFilepathsToEachChannelOfEachPatientValidation,
+                             listOfFilepathsToGtLabelsOfEachPatientValidationOnSamplesAndDsc,
+                             listOfFilepathsToRoiMaskOfEachPatientValidation,
+                             paths_to_wmaps_per_sampl_cat_per_subj_val,
+                             pad_input_imgs,
+                             None,  # no augmentation in val.
+                             None)  # no augmentation in val.
+
     sampling_job_submitted_train = False
     sampling_job_submitted_val = False
     # For parallel extraction of samples for next train/val while processing previous iteration.
     worker_pool = None
-    if num_parallel_proc_sampling > -1 : # Use multiprocessing.
-        worker_pool = ThreadPool(processes=1) # Or multiprocessing.Pool(...), same API.
-    
+    if num_parallel_proc_sampling > -1:  # Use multiprocessing.
+        worker_pool = ThreadPool(processes=1)  # Or multiprocessing.Pool(...), same API.
+
     try:
         model_num_epochs_trained = trainer.get_num_epochs_trained_tfv().eval(session=sessionTf)
-        while model_num_epochs_trained < n_epochs :
+        while model_num_epochs_trained < n_epochs:
             epoch = model_num_epochs_trained
-            
-            acc_monitor_for_ep_train = AccuracyOfEpochMonitorSegmentation(log, 0, model_num_epochs_trained, cnn3d.num_classes, num_subepochs)
+
+            acc_monitor_for_ep_train = \
+                AccuracyOfEpochMonitorSegmentation(log, 0,
+                                                   model_num_epochs_trained,
+                                                   cnn3d.num_classes,
+                                                   num_subepochs,
+                                                   tensorboard_logger=tensorboard_loggers['train']
+                                                   if tensorboard_loggers is not None else None)
```

Comment:
Kamnitsask 16 seconds ago Member

This if here inside the arguments, in its own line, reads "very" weird.

If we rename "tensorboard_loggers" to "tb_loggers" it can go to same line probably.

Reply:
once the AccuracyOfEpochMonitorSegmentation fucntion has been renamed to something smaller this issue will be easily mitigated